src/guide/jbs-jdk-bug-system.md

```diff
@@ -176,36 +146,38 @@ style ClosedIncomplete fill:lightgreen
 
 ## Triaging an issue
 
-For most OpenJDK groups Triage is performed on a regular basis (at least weekly) by triage teams. Each triage team consists of contributors who are area experts for that group of issues. If you haven't been selected to be part of a triage team for a specific area you shouldn't be triaging bugs in that area.
+For most JDK areas, triage is performed on a regular basis (at least weekly) by triage teams. Each triage team consists of contributors who are area experts for a specific area or feature. If you haven't been selected to be part of a triage team for a specific area *you shouldn't be triaging bugs in that area*.
```

Comment:
I'm wondering if there should be an exception for a bug that a Committer is claiming for themselves.  Perhaps the triage teams are so large that there is no need to declare an exception?

Reply:
In practice a lot of people triage their own bugs. I'm not sure if this is a good thing or not. On one hand it reduces the work for the triage team, but on the other hand we miss the verification step to make sure the bug is correctly filed. I don't think we should explicitly say that this is ok here in the guide, but each area could on an individual basis decide that experienced engineers in the team are "part of the triage team" and have the mandate to do this.